rr_platform/src/motor_relay_node/motor_relay_node.cpp

```diff
@@ -1,15 +1,22 @@
 #include <ros/ros.h>
 #include <rr_platform/speed.h>
 #include <rr_platform/steering.h>
+#include <rr_platform/chassis_state.h>
 #include <boost/asio.hpp>
 
-int desiredSpeed = 0;
-int desiredSteer = 0;
-int prevAngle = 0;
-int prevSpeed = 0;
+double desiredSpeed = 0;
+double desiredSteer = 0;
+double prevAngle = 0;
+double prevSpeed = 0;
+
+double p_const = 0;
```

Comment:
Using const in the name here may be a bit misleading. I understand what you're going for, but const has specific meaning in C++. A very common naming convention for these coefficients is Kp, Ki, and Kd.